spring-petclinic-microservices/pom.xml

```diff
@@ -3,228 +3,228 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <parent>
+  <parent>
```

Comment:
It's not easy to review this file.

Reply:
just reformat the pom, in intellj view it doesn't show the diff , but it shows on github.